pkg/kn/commands/channel/flags.go

```diff
@@ -120,3 +125,64 @@ func printChannelListWithNamespace(channelList *messagingv1beta1.ChannelList, op
 
 	return append(rows, others...), nil
 }
+
+// ListTypesHandlers handles printing human readable table for `kn channel list-types`
+func ListTypesHandlers(h printers.PrintHandler) {
+	channelTypesColumnDefinitions := []metav1beta1.TableColumnDefinition{
+		{Name: "Type", Type: "string", Description: "Kind / Type of the channel type", Priority: 1},
+		{Name: "Group", Type: "string", Description: "Group of the channel type", Priority: 1},
+		{Name: "Version", Type: "string", Description: "Version of the channel type", Priority: 1},
```

Comment:
I can comment all these lines (131-134) and all tests are passing... so there is no coverage which agrees with the gocov error in CI.

Also, lines 37-43 are same which might already been an issue before your PR.